js/modules/loki_app_dot_net_api.js

```diff
@@ -1256,6 +1289,15 @@ class LokiPublicChannelAPI {
     if (data.counts && Number.isInteger(data.counts.subscribers)) {
       this.conversation.setSubscriberCount(data.counts.subscribers);
     }
+
+    await window.Signal.Data.updateConversation(
```

Comment:
Does `this.conversation.set` not write to sqlite?

Reply:
No, i think it only writes to the backbone model.